03/app.js

```diff
@@ -43,10 +65,42 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
+    if (typeof this[isbn] === 'undefined') {
+        // taki ISBN nie istnieje w moim spisie
+        return null;
+    }
+
+    const title = this[isbn]['title'][lang];
+
+    if (lang === 'pl') {
```

Comment:
Tutaj nie chodzi, aby sprawdzić czy `lang` jest równe `pl` tylko czy istnieje taki element w tablicy więc należałoby sprawdzić czy `if(this[isbn]['title'][lang])`.

Ps. Jeżeli mielibyśmy 100 języków to tworzylibyśmy 100 instrukcji if? Jeśli doszły by kolejne 100 to znów byśmy to w ten sposób zapisywali? Musimy tworząc kod patrzeć w przyszłość i przewidywać pewne konsekwencje ;)

Reply:
uf, na razie "nie widzę tego" (a wiem że powinienem)